tests/dynamic_data/publications/publication_name_test.py

```diff
@@ -62,9 +63,8 @@ def pytest_generate_tests(metafunc):
                     (publication, method_name),
                 ])
                 ids.append(testcase_id)
-    publ_type_name = cls.publication_type.split('.')[-1]
     metafunc.parametrize(
```

Comment:
if cls.publication_type is None, set publ_type_name to "publication", otherwise keep current behavior